README.adoc

```diff
@@ -137,22 +136,21 @@ The `handles()` method inspects the HTTP response code to determine whether an e
 
 Now, instantiate the `SystemClient` interface and use it in the `inventory` service. If you want to connect only with the default host name, you can easily instantiate the `SystemClient` with CDI annotations. CDI injection simplifies the process of bootstrapping the client.
 
-First, you need to tell the MicroProfile Rest Client API the base URL value for the `system` service endpoint.
-You can configure the default base URL of the `SystemClient` instance by using MicroProfile Config. The feature has been enabled for you in both `server.xml` and `pom.xml` files.
+First, you need to define the base URL of the `SystemClient` instance.
+You can configure the default base URL by using MicroProfile Config. The feature has been enabled for you in both `server.xml` and `pom.xml` files.
```

Comment:
How about, "Configure the default base URL with the MicroProfile Config feature. This feature is enabled for you in both..."